aten/src/ATen/nn.yaml

```diff
@@ -5,6 +5,12 @@
   scalar_check:
     output: 'false'
     grad_input: 'false'
+  CPU:
```

Comment:
if this is supposed to demonstrate the usage, can you just comment to that effect?

Reply:
Added